src/main/java/no/finn/unleash/util/UnleashConfig.java

```diff
@@ -232,6 +248,7 @@ public boolean isProxyAuthenticationByJvmProperties() {
         private @Nullable UnleashScheduledExecutor scheduledExecutor;
         private @Nullable UnleashSubscriber unleashSubscriber;
         private boolean isProxyAuthenticationByJvmProperties;
+        private Strategy fallbackStrategy;
```

Comment:
If we initialize it to to `UNKNOWN_STRATEGY` we can avoid the null checks on line 83:86.

Reply:
We need a null check at one point to endure the system can't be configured in a bad state. So I can move the null check to the builder setter. 

It pretty common to use null as unsetting a value and saves the user from needing to know what the unknown option is. However if you would prefer I skip that check, I can remove it.

Reply:
tbh, I do not think it matters much. lets just keep it.